readme.md

```diff
@@ -139,7 +188,16 @@ Define a Javascript constructor called 'Instructor'. Every instance of Instructo
 Instantiate an instructor named 'Andy' and call its `receivePresent` method with "Resin Laying Deer Figurine, Gold" as the argument.
 
 ```js
-// Your code starts here...
+
+var instructor = function(name, gift){
+  this.name = name
+  this.gift = gift
+  this.recievePresent = function(){
+      console.log(name + "drops on the floor"+ gift)
+    }
+  }
+
+Instructor("andy", "Resin Laying Deer Figurine, Gold")
```

Comment:
We moved the "gift" argument to "receivePresent", so the instructor can receive different presents.